cunumeric/module.py

```diff
@@ -195,6 +195,11 @@ def array(obj, dtype=None, copy=True, order="K", subok=False, ndmin=0):
     return array
 
 
+@copy_docstring(np.choose)
+def choose(a, choices, out=None, mode="raise"):
+    return a.choose(choices=choices, out=out, mode=mode, stacklevel=2)
```

Comment:
Make sure to convert the input to a cunumeric ndarray using `ndarray.convert_to_cunumeric_ndarray`

Reply:
Yes!